### PR TITLE
Complete run-through, cosmetic updates

### PR DESCRIPTION
**Pull request by zenodhaene**

### CODE REVIEW

File: hackathon/activities/containerlab-b-start/README.md [hunk 1]
```diff
@@ -102,7 +102,9 @@ sudo docker image list
 ```
 ```
 REPOSITORY                                 TAG         IMAGE ID       CREATED              SIZE
-vrnetlab/vr-sros                           24.3.R3     36a2e22c7267   About a minute ago   766MB
+...
+vrnetlab/nokia_vsr                         24.3.R3     1b8042dbbb41   11 seconds ago       767MB
```
Comment (hellt) [inline hackathon/activities/containerlab-b-start/README.md:106]: mm, let me check this, I think I need to set the image type to `sros` so that the image name would be `nokia_sros` and not `nokia_vsr`

File: hackathon/activities/srlinux-b-streaming-telemetry/README.md [hunk 2]
```diff
@@ -90,8 +90,6 @@ From the output you can find that for us to retrieve the *host-name* we will nee
 
 The YANG browser lets you do a search through the SR Linux YANG model. By simply typing for certain keywords, the YANG browser will return any matches found.
 
-<img width="708" alt="yangbrowser" src="https://github.com/jgcumming/srx2024-hackplanning/assets/33862202/ab13e924-7f2d-4750-b7a9-6841c2e9b2fc">
```
Comment (hellt) [inline hackathon/activities/srlinux-b-streaming-telemetry/README.md:93]: did you remove this because the picture is outdated?

Comment (zenodhaene) [reply hackathon/activities/srlinux-b-streaming-telemetry/README.md:93]: It's not clear what the picture is supposed to be. The link in the src argument is not a public repository.

**Comment by hellt** [comment]
I will bring in those changes into planning repo

File: hackathon/activities/sros-a-event-handling/README.md [hunk 3]
```diff
@@ -504,7 +504,7 @@ For the example solution, we re-use the configuration from task 1, amending it w
...
 6. Play with the `admin-state` attribute on the BGP session to the route-reflector on `pe2` in order to simulate a failure and the restoration of the BGP session.
 
-Mind the `admin-state` configuration for the various configurations being added as some are disabled by default. For possible help and inspiration, an example solution is provided in [examples/bgp-opergroup.py](./examples/opergroup.py). This file includes a validated `log` configuration matching what is outlined above. If you are working on the file source code, ensure it is reloaded into the `python-script` object in the router's memory using `/perform python python-script reload script bgp-opergroup`.
+Mind the `admin-state` configuration for the various configurations being added as some are disabled by default. For possible help and inspiration, an example solution is provided in [examples/bgp-opergroup.py](./examples/bgp-opergroup.py). This file includes a validated `log` configuration matching what is outlined above. If you are working on the file source code, ensure it is reloaded into the `python-script` object in the router's memory using `/perform python python-script reload script bgp-opergroup`.
```
Comment (hellt) [inline hackathon/activities/sros-a-event-handling/README.md:507]: @jgcumming I carried over the edits Zeno did for srl exercises, but I haven't touched sros ones.

Please check if you want to pull them in

File: hackathon/activities/srlinux-i-custom-cli/solutions/skeleton.py [hunk 4]
```diff
@@ -13,55 +9,56 @@
...
     '''
         _populate_schema() method: fill in schema from state datastore
+        Input: state_datastore, state datastore extract
         Input: arguments, the CLI command's context
-        Return: filled in schema 
-    ''' 
-    def _populate_schema(self, arguments):
+        Return: filled in schema
+    '''
+    def _populate_schema(self, state_datastore, arguments):
```
Comment (hellt) [inline hackathon/activities/srlinux-i-custom-cli/solutions/skeleton.py:44]: @zenodhaene why did you add `state_datastore` here?

Comment (zenodhaene) [reply hackathon/activities/srlinux-i-custom-cli/solutions/skeleton.py:44]: There is an inline copy of this file in the README of this activity, but the two didn't align. I simply copied the inline version (which is confirmed working) to the attached file in the solutions folder.

File: hackathon/activities/containerlab-b-start/README.md (hunk 1 above)
Comment (hellt) [reply hackathon/activities/containerlab-b-start/README.md:106]: fixed vrnetlab to use nokia_sros image name
